homework_7/animals/Animal.java

```diff
@@ -0,0 +1,58 @@
+package animals;
+
+public class Animal {
+    private String nameAnimal;
+    private static int animalsCount;
```

Comment:
Class' members should have a specific order:
1. constants
2. static variables
3. instance variables
4. constructors
5. getters + setters
6. public methods
7. private methods
Please change this class' members order according to the info in this comment.